### PR TITLE
Add compression job performance hotfix instructions

### PR DESCRIPTION
**Pull request by ramonguiu**
## Description

Provide instructions to apply hotfix for compression job slow performance to Promscale

### CODE REVIEW

File: docs/compression-job-hotfix.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+those functions by executing SQL commands against the databae. This problem
+affects Promscale version 0.10.0 and earlier.
+
+Below we explain the steps to apply a hotfix to your Promscale instance. This 
+fix will be included in the upcoming release of Promscale.
+
+## Steps to Apply the Hotfix
+
+1. Upgrade Promscale to version 0.10.0.
+2. Download the [hotfix file](scripts/compression-job-performance-hotfix.sql).
```
Comment (JamesGuthrie) [inline docs/compression-job-hotfix.md:17]: The filename has been changed

File: docs/compression-job-hotfix.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+3. Apply the hotfix by executing the SQL code in the hotfix. For example you can
+use psql: `psql <your connection options> -f compression-job-performance-hotfix.sql`
```
Comment (JamesGuthrie) [inline docs/compression-job-hotfix.md:19]: here too

File: docs/compression-job-hotfix.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## Rollback the Hotfix
+
+To rollback the hotfix and restore the original version of the functions in 
+Promscale 0.10.0:
+
+1. Download the [rollback file](scripts/rollback-compression-job-performance-hotfix.sql).
```
Comment (JamesGuthrie) [inline docs/compression-job-hotfix.md:27]: here too

File: docs/compression-job-hotfix.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+3. Apply the rollback file by executing the SQL code in the hotfix:
+`psql <your connection options> -f rollback-compression-job-performance-hotfix.sql`
```
Comment (JamesGuthrie) [inline docs/compression-job-hotfix.md:29]: and here

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

**Review by antekresic — COMMENTED** [review]
Markdown looks good, letting other guys to review the sql script.

File: docs/compression-job-hotfix.md (hunk 1 above)
Comment (ramonguiu) [reply docs/compression-job-hotfix.md]: I had changed the file and forgot to save before committing ...

**Review by sumerman — APPROVED** [review]

**Review by JamesGuthrie — APPROVED** [review]
LGTM

**Pull request merged**